src/build-tx/coinselect-lib/inputs/accumulative.js

```diff
@@ -13,6 +13,36 @@ export default function accumulative(utxos, outputs, feeRate, options) {
     const inputs = [];
     const outAccum = utils.sumOrNaN(outputs);
 
+    // split utxos into required and the rest
+    const requiredUtxo = [];
```

Comment:
Let's use "s" at the end for consistency.

```suggestion
    const requiredUtxos = [];
```

Reply:
addressed d6a96bd4d0b8544ffe3e703b47e44a9fe3783e17